_assets/scripts/run_functional_tests.sh

```diff
@@ -24,29 +24,36 @@ mkdir -p "${merged_coverage_reports_path}"
 mkdir -p "${test_results_path}"
 
 all_compose_files="-f ${root_path}/docker-compose.anvil.yml -f ${root_path}/docker-compose.test.status-go.yml"
+timestamp=$(date +%s)
 
 # Run functional tests
 echo -e "${GRN}Running tests${RST}, HEAD: $(git rev-parse HEAD)"
-docker-compose ${all_compose_files} up -d --build --remove-orphans
+docker-compose -p ${timestamp} ${all_compose_files} up -d --build --remove-orphans
 
 echo -e "${GRN}Running tests-rpc${RST}" # Follow the logs, wait for them to finish
-docker-compose ${all_compose_files} logs -f tests-rpc > "${root_path}/tests-rpc.log"
+docker-compose -p ${timestamp} ${all_compose_files} logs -f tests-rpc > "${root_path}/tests-rpc.log"
```

Comment:
You should be using Docker Compose V2 and not V1, by using `docker compose` instead:
https://docs.docker.com/compose/releases/migrate/#what-are-the-differences-between-compose-v1-and-compose-v2

But it's probably worth doing this in a separate PR.